### ISSUE TITLE
Can we free the editor from the narrow left hand column

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/css_editor#2]
The css editor window is very much cramped into a narrow column on the left side of the configuration page. I seem to recall that this is a larger issue that effects all theme config. 

Is there anything we can do about this in the module or does this require a change in core?

![makewider](https://user-images.githubusercontent.com/3144571/132116756-2df68e6d-9b86-44ce-9cea-a152f713f618.png)

**Comment by olafgrabienski** [comment]
> I seem to recall that this is a larger issue that effects all theme config.

Right, the reason for the narrow theme settings section is probably the color preview beside the color settings. The class `.has-preview` on the `system-theme-settings` form sets the width to 20em (usually 320px). In addition, the `css-editor-textarea` textarea is hardcoded to a width of 319px.

(Didn't find a respective core issue so far.)

**Comment by albanycomputers** [comment]
Agreed...  Making it wider would certainly improve usability. :-)
